Index: linux-4.9.37/drivers/char/tpm/Kconfig
===================================================================
--- linux-4.9.37.orig/drivers/char/tpm/Kconfig
+++ linux-4.9.37/drivers/char/tpm/Kconfig
@@ -44,6 +44,7 @@ config TCG_TIS
 config TCG_TIS_SPI
 	tristate "TPM Interface Specification 1.3 Interface / TPM 2.0 FIFO Interface - (SPI)"
 	depends on SPI
+	default n
 	select TCG_TIS_CORE
 	---help---
 	  If you have a TPM security chip which is connected to a regular,
